aspnetcore/tutorials/your-first-mac-aspnet.md

```diff
@@ -1,81 +1,71 @@
 
 ```console
-yo aspnet
-   ```
-
-* Select **Web Application Basic [without Membership and Authorization]** and tap Enter
-
-* Select Bootstrap (3.3.6) as the UI framework and tap Enter
+dotnet new mvc
+```
 
-* Use "MyFirstApp" for the app name and tap Enter
-
-When the generator completes scaffolding the files, it will instruct you to restore, build, and run the application.
+When the CLI command completes; the following output and files are produced.
 
 ```console
-Your project is now created, you can use the following commands to get going
-       cd "MyFirstApp"
-       dotnet restore
-       dotnet build (optional, build will also happen with it's run)
-       dotnet run
-   ```
-
-## Developing ASP.NET Core Applications on a Mac With Visual Studio Code
+Content generation time: 79.6691 ms
+The template "Empty ASP.NET Core Web Application" created successfully.
+```
 
-* Start **Visual Studio Code**
+* Startup.cs : [Startup Class](../fundamentals/startup.md) - class configures the request pipeline that handles all requests made to the application.
+* Program.cs : [Program Class](../fundamentals/) that contains the Main entry point of the application.
+* firstmacapp.csproj : [Project file](https://docs.microsoft.com/en-us/dotnet/articles/core/preview3/tools/csproj) MSBuild Project file format for ASP.NET Core applications. Contains Project to Project references, NuGet References and other project related items.
+* appsettings.json / appsettings.Development.json : Environment base app settings configuration file. [See Configuration](../fundamentals/configuration).
+* bower.json : Bower package dependencies for the project.
+* .bowerrc : Bower configuration file which defines where to install the components when Bower downloads the assets.
+* bundleconfig.json : configuration files for bundling and minifying front-end JavaScript and CSS assets.
+* Views : Contains the Razor views for the Controllers as well as Shared Views. Views are the components that display the app's user interface (UI). Generally, this UI displays the model data.
+* Controllers : Contains MVC Controllers, initially _HomeController.cs_. Controllers are Classes that handle browser requests, retrieve model data, and then specify view templates that return a response to the browser.
+* wwwroot : Web application root folder.
 
-* Tap **File > Open** and navigate to your ASP.NET Core app
+For more on the MVC pattern see [What is the MVC pattern?](../mvc/overview).
 
-![File menu](your-first-mac-aspnet/_static/file-open.png)
+## Developing ASP.NET Core applications on a Mac with Visual Studio Code
 
-When the application is opened, Visual Studio Code will prompt to restore the needed project dependencies as well as add build and debug dependencies.
+Select the _Program.cs_ file and Visual Studio Code will prompt to restore the needed project dependencies as well as add build and debug dependencies.
```

Comment:
@spboyer VS Code Markdown editor uses underscores for italics - is there a way to make it use asterisks?